ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/ImagesTableContainer.tsx

```diff
@@ -22,6 +22,7 @@ type ImagesTableContainerProps = {
     cveStatusTab?: CveStatusTab; // TODO Make this required once Observed/Deferred/FP states are re-implemented
     pagination: ReturnType<typeof useURLPagination>;
     onWatchImage: ImagesTableProps['onWatchImage'];
+    onUnwatchImage: ImagesTableProps['onUnwatchImage'];
```

Comment:
I could probably refactor these into a single callback/action flow, but the two option approach is simpler and more straightforward IMO.